lib/lookup-app-by-phone-number.js

```diff
@@ -60,14 +60,6 @@ async function lookupAppByPhoneNumber(pool, logger, phoneNumber, voip_carrier_si
     return obj;
   }
 
-  /* check if this gateway routes all calls to an application */
-  if (voip_carrier_sid) {
-    const [r2] = await pp.query(
-      'SELECT application_sid from voip_carriers where voip_carrier_sid = ?', voip_carrier_sid);
-    if (r2.length > 0 && r2[0].application_sid) {
-      return await lookupAppBySid(r2[0].application_sid);
-    }
-  }
 
   // if not, check the regex patterns that have been set up
```

Comment:
The "if" part of the comment no longer makes sense, since you moved the condition above down below.  Can you rewrite the comments to clarify the new implementation?